docs/v2-strategies/walkthrough.md

```diff
@@ -0,0 +1,218 @@
+
+
+### Run the script 
+
+[![controller](../diagrams/22.png)](../diagrams/22.png)
+
+```shell
+start --script v2_simple_directional_rsi.py --conf conf_v2_simple_directional_rsi_1.yml
+```
+
+The bot should now be running and start placing orders for both pairs. Run the **status** command to see the bot status.
+
+```
+status --live
+```
+
+---
+
+## Strategies V2 + Controller(s)
+
+![advanced](diagrams/11.png)
+
+
+In this scenario, let's say we want to create a single bot that manages two distinct trading pairs on **Binance Perpetual**, each configured with unique buy and sell spreads. To achieve this, we can utilize the [**v2_generic_with_controllers.py**](https://github.com/hummingbot/hummingbot/blob/development/scripts/v2_generic_with_controllers.py) script alongside the [**pmm_simple.py**](https://github.com/hummingbot/hummingbot/blob/development/controllers/market_making/pmm_simple.py) controller. The initial step involves generating a separate controller configuration for each trading pair.
+
+
+### Create the controller configs
+
+[![controller](../diagrams/15.png)](../diagrams/15.png)
+
+```shell
+create --controller-config market.making.pmm_simple
+```
+
+```shell
+
+Enter the name of the exchange to trade on >> binance_perpetual
+Enter the name of the trading pair to trade on >> WLD-USDT
+Enter the total amount in quote asset to use for trading >> 100
+Enter a comma-separated list of buy spreads >> 0.01, 0.02
+Enter a comma-separated list of sell spreads >> 0.01, 0.02
+Enter the refresh time in seconds for executors >> 20
+Set the leverage to use for trading >> 20
+Enter the stop loss >> 0.03
+Enter the take profit >> 0.02 
+Enter the time limit in seconds >> 2700
+Enter the order type for taking profit >> LIMIT
+Enter the trailing stop as activation_price, trailing_delta >> 0.013, 0.003
+Enter a file name for your configuration >> conf_market_making.pmm_simple_1.yml
+
+```
+
+This will create the **conf_market_making.pmm_simple_1.yml** controller config file under the **conf/controllers** folder 
+
+Repeat the steps to create a new controller config, but this time, use **MYRO-USDT** as the trading pair. 
+
+Also, adjust the buy and sell spreads to **0.02, 0.05**, and **0.02, 0.05**, respectively. Save this modified configuration under the file name **conf_market_making.pmm_simple_2.yml**
+
+You should now have two controller config files: 
+
+```shell
+conf_market_making.pmm_simple_1.yml
+conf_market_making.pmm_simple_2.yml
+```
+
+### Create the generic script config
+
+[![script-config](../diagrams/16.png)](../diagrams/16.png)
+
+```
+create --script-config v2_generic_with_controllers
+
+```
+
+
+
+```python
+Enter controller configurations >>> conf_market_making.pmm_simple_1.yml, conf_market_making.pmm_simple_2.yml
+Enter a new file name for your configuration >>> conf_v2_generic_with_controllers_1.yml
+
+```
+
```

Comment:
Add a tip telling the user that it might be easier for them to edit this generic file and replace it with new controller configs than re-generate each time.